Model/AutoRoute.php

```diff
@@ -50,7 +50,7 @@ public function setType($type)
         $this->setDefault('type', $type);
     }
 
-    public function setRedirectTarget(AutoRouteInterface $redirectRoute)
+    public function setRedirectTarget($redirectRoute)
```

Comment:
Ah yeah. I have just created a PR to fix that: https://github.com/symfony-cmf/RoutingAutoBundle/pull/166